cmd/gardener-scheduler/main.go

```diff
@@ -18,24 +18,23 @@ import (
 	"fmt"
 	"os"
 	"runtime"
-	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
 
 	"github.com/gardener/gardener/cmd/gardener-scheduler/app"
 	"github.com/gardener/gardener/cmd/utils"
 	"github.com/gardener/gardener/pkg/controllermanager/features"
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
 )
 
-func init() {
-	features.RegisterFeatureGates()
-}
-
 func main() {
 	if len(os.Getenv("GOMAXPROCS")) == 0 {
 		runtime.GOMAXPROCS(runtime.NumCPU())
 	}
 
+	features.RegisterFeatureGates()
```

Comment:
Why do you move this from `init()` to here and why only for the scheduler?

Reply:
Why does it needs `init()` in the first place when it was in the `main` package.

> why only for the scheduler?

I was fixing a deprecation warning (package change)

Reply:
I'm not opposing in general, just voting for doing it in a consistent way.
Do you mind adjusting the other places as well?

Reply:
Already done